src/IO/H5/StellarCollapseEos.hpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::StellarCollapseEos
+
+#pragma once
+
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "DataStructures/BoostMultiArray.hpp"
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ * \brief Reads in tabulated equation of state file from stellarcollapse.org
+ *
+ * Reads in H5 file containing data for tabulated equation of state.
+ * Contains functions to obtain thermodynamic quantities from the file,
+ * stored as either rank-1 or rank-3 datasets.
+ *
+ * It is assumed that the file is in the format of the SRO
+ * (Schneider, Roberts, Ott 2017) Equation of State files at
+ * <a href="https://stellarcollapse.org/SROEOS">
+ * stellarcollapse.org </a>
+ *
+ * The description of each dataset in the file can be found
+ * [here](https://bitbucket.org/andschn/sroeos/src/master/
+User_Guide/User_Guide.pdf?fileviewer=file-view-default)
+ *
+ */
+class StellarCollapseEos : public h5::Object {
+ public:
+  /// \cond HIDDEN_SYMBOLS
+  // The stellar collapse files do not have an extension when they are at the
+  // root
+  // level ('/') and we currently do not have any combined stellar collapse
```

Comment:
Can you reduce the number of lines this comment takes because line 40 only has one thing on it.